dissect/target/plugins/os/unix/log/auth.py

```diff
@@ -241,15 +253,23 @@ def build_record(self, ts: datetime, source: Path, service: str, pid: int, messa
             ("string", "message"),
         ]
 
-        record_values = {}
-        record_values["ts"] = ts
-        record_values["source"] = source
-        record_values["service"] = service
-        record_values["pid"] = pid
-        record_values["message"] = message
-        record_values["_target"] = self.target
-
-        for key, value in self._parse_additional_fields(service, message).items():
+        record_values = {
+            "ts": ts,
+            "message": line,
+            "service": None,
+            "pid": None,
+            "source": source,
+            "_target": self.target,
+        }
+
+        match = RE_LINE.match(line)
+        if match:
+            values = match.groupdict()
+            del values["ts"]
+            values["message"] = values["message"].strip()
```

Comment:
Here I would suggest to do the stripping actions within the regex, such that this additional line is not needed. Something along the lines of `\s*?` before and after the `message` capturing group should work